src/cloud/getting-started/quick-start.md

```diff
@@ -0,0 +1,89 @@
+# Quick Start
+
+This guide will help you get started with PGVecto.rs Cloud. First, you need to sign up for a PGVecto.rs Cloud account. If you haven't done so already, you can sign up for a free account [here](../getting-started/sign-up.md).
+
+## Create a cluster
+
+::: tip
+You can only create Enterprise plan after you bind a [credit card](../payment/credit-card.md).
+:::
+
+To create a new cluster, navigate to the PGVecto.rs Cloud Console and click on the "Create Cluster" button.
+
+![](../images/after_login_in.png)
+
+You need to specify the following information:
+- **Cluster Name**: A unique name for your cluster.
+- **Plan**: The plan you want to use for your cluster, for information about Cloud's paid plan options, see [Cloud plans](../pricing/price-plan).
+- **Cloud Provider**: The cloud provider where your cluster will be deployed.
+- **Region**: The region where your cluster will be deployed.
+- **Database Name**: The name of the database that you will store vector data in. 
+
+![](../images/create_cluster.png)
+
+## Cluster Info 
+
+After creating a cluster, you will be directed to the cluster info page. Here you can find the details of your cluster, such as the connection string, the status of the cluster, and resource usage of the cluster.
+
+![](../images/cluster_info.png)
+
+We provide two types of connection endpoints:
+- Super User Endpoint: This connection string is used to connect to the cluster as a superuser(postgres). You can use this connection string to create databases, users, and manage the cluster.
+- Vector User Endpoint: This connection string is used to connect to the cluster as a vector user. You can use this connection string to store and query vector data.
+
+![](../images/two_types_connections.png)
```

Comment:
Duplicated to the [connect-with-psql.md]?

Reply:
Fixed.